source/create_synonyms_and_grants_for_user.sql

```diff
@@ -108,6 +108,7 @@ create or replace synonym &ut3_user..ut_runner for &&ut3_owner..ut_runner;
 create or replace synonym &ut3_user..ut_teamcity_reporter for &&ut3_owner..ut_teamcity_reporter;
 create or replace synonym &ut3_user..ut_xunit_reporter for &&ut3_owner..ut_xunit_reporter;
 create or replace synonym &ut3_user..ut_documentation_reporter for &&ut3_owner..ut_documentation_reporter;
+create or replace synonym &ut3_user..ut_ansiconsole_helper for &&ut3_owner..ut_ansiconsole_helper;
```

Comment:
I would remove this synonym. ut_ansiconsole_helper is `internal` code and we don't need to make it publicly available.
Not exposing it publicly gives us more freedom to change internals of the project.